pkg/features/list.go

```diff
@@ -6,4 +6,7 @@ var (
 
 	// PrintCentralUpdateDiff enables printing the diff of the central update
 	PrintCentralUpdateDiff = registerFeature("Print the diff of the central update", "RHACS_PRINT_CENTRAL_UPDATE_DIFF", false)
+
+	// UseOperatorsConfigMap makes Fleetshard-sync service use ACS Operators configuration from ConfigMap. It is useful for E2E testing
+	UseOperatorsConfigMap = registerFeature("Use ACS Operators configuration from ConfigMap", "RHACS_USE_OPERATORS_CONFIGMAP", false)
```

Comment:
nit: Keeping the name a bit more general to debug mode gives more flexibility later, i.e. to use to mock fleet-manager API responses.

```suggestion
	UseOperatorsConfigMap = registerFeature("Use ACS Operators configuration from ConfigMap", "RHACS_DEBUG_MODE", false)
```

Reply:
I think you not had the time since our sync, commenting here for visibility to change the name.